tests/data/test_preprocessing.py

```diff
@@ -21,7 +18,7 @@ def test_remove_consecutive_interactions():
     schema += Schema([ColumnSchema.create_continuous("timestamp", tags=[Tag.SESSION])])
 
     interactions_df = generate_item_interactions(500, schema)
-    filtered_df = remove_consecutive_interactions(interactions_df.copy())
+    filtered_df = preprocessing.remove_consecutive_interactions(interactions_df.copy())
```

Comment:
my was throwing an error about the module not having the attribute with the function import version (not sure I fully understand why). But this seems to work without error:

```
tests/data/test_preprocessing.py:5: error: Module "transformers4rec.data.preprocessing" has no attribute "add_item_first_seen_col_to_df"  [attr-defined]
tests/data/test_preprocessing.py:5: error: Module "transformers4rec.data.preprocessing" has no attribute "remove_consecutive_interactions"  [attr-defined]
```